### DISCUSSION TITLE
Docker Swarm support?

### DISCUSSION THREAD

**Discussion by adamshand** [lllllllillllllillll/DweebUI#81] [General]
Hey,

Looks like a cool project, thanks for creating!  

I'm wondering if you have any interest in eventually supporting Docker Swarm?

Thanks!
Adam.

**Comment by lllllllillllllillll** [comment]
At some point. 
It's kinda already built-in with the nodejs module [dockerode](https://github.com/apocas/dockerode) from [apocas](https://github.com/apocas). I just need to build the UI for it.

**Reply by adamshand** [reply]
Ooh, thanks for the link to dockernode, didn't know about that!  

Look foward to testing swarm support, thanks!
